### DISCUSSION TITLE
all emails don't sent

### DISCUSSION THREAD

**Discussion by david0003** [postalserver/postal#1451] [Help with using Postal]
The server was running fine... all of a sudden it stopped working. When I test it, I find that port 25 is open. But Emails Stay pending.
This issue occurred after I sent a bunch of emails at once. After that, the server did not send any e-mail .... although after I tested it on more than one site, the result was that it works normally.
there is the out put of postal status
![Capture d’écran 2021-06-27 121211](https://user-images.githubusercontent.com/85866422/123558411-66852480-d796-11eb-8484-551ca23a6ad0.jpg)

**Comment by willpower232** [comment]
The low time value mentioned in your screenshot implies that Postal was restarted recently. You may need to use the database directly to free any messages which are locked to a queue process that no longer exists.

**Comment by david0003** [comment]
> The low time value mentioned in your screenshot implies that Postal was restarted recently. You may need to use the database directly to free any messages which are locked to a queue process that no longer exists.

How I do That?

**Comment by willpower232** [comment]
Using a database program you can look at the database directly and read this to get a steer

https://github.com/postalhq/postal/issues/1119#issuecomment-643985608
